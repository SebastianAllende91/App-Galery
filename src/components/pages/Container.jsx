import React from "react";
import Cards from "../layouts/Cards";

const Container = () => {
  return (
    <div aria-label="container" className="container border my-2">
      <Cards />
    </div>
  );
};

export default Container;
