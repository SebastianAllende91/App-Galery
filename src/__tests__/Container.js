import { render, screen } from "@testing-library/react";
import Container from "../components/pages/Container";

describe("<Container/>", () => {
  it("Snapshot principal", () => {
    render(<Container />);

    expect(screen.getByLabelText("container")).toMatchSnapshot();
  });
});
