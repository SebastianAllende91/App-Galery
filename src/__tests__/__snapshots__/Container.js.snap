// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container/> Snapshot principal 1`] = `
<div
  aria-label="container"
  class="container border my-2"
>
  <div
    class="my-2"
  >
    <div>
      <form>
        <label
          class="w-100"
          style="font-size: 20px;"
        >
          Search: 
          <input
            class="me-3"
            name="inputText"
            type="text"
          />
          <button
            class="btn btn-warning"
            style="height: 31px;"
            type="submit"
          >
            <ion-icon
              name="search"
            />
          </button>
        </label>
      </form>
    </div>
    <div
      class="alert alert-danger"
      role="alert"
      style="margin-top: 150px; margin-bottom: 150px;"
    >
      No images found!!!
    </div>
    <div
      class="d-flex align-items-center"
    >
      <strong>
        Loading...
      </strong>
      <div
        aria-hidden="true"
        class="spinner-border ms-auto"
        role="status"
      />
    </div>
    <div
      aria-label="imagenes"
      class="row my-2"
    />
  </div>
</div>
`;
